srml/contract/src/lib.rs

```diff
@@ -147,8 +154,43 @@ decl_module! {
 	/// Contracts module.
 	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
 		fn deposit_event<T>() = default;
-		// TODO: Change AccountId to staking::Address
-		/// Make a call to a specified account, optionally transferring some balance.
+
+		/// Updates the schedule for metering contracts.
+		///
+		/// The schedule should have a greater version than the stored schedule.
```

Comment:
should implies this wouldn't fail (which it actively does) if that condition isn't met. I prefer the usage of `must` in these cases as it is stronger and thus more explicit on that it will not accept any other condition.

Reply:
Thank you for the explaination!